DatadogInternal/Sources/Telemetry/Telemetry.swift

```diff
@@ -24,12 +24,6 @@ internal protocol Telemetry {
     ///   - kind: The kind of error.
     ///   - stack: The stack trace.
     func error(id: String, message: String, kind: String?, stack: String?)
-
-    /// Sends a `TelemetryConfigurationEvent` event.
-    ///
-    /// - Parameters:
-    ///   - configuration: The current configuration
-    func configuration(configuration: FeaturesConfiguration)
```

Comment:
question/ How does configuration telemetry work now? I struggled with finding corresponding changes, only noticed disabled test 🤔💭.

Reply:
The `Telemetry` protocol does not require this method, but the `RUMTelemetry` does expose it. The reason being that the `FeaturesConfiguration` type is not part of the `DatadogInternal`, it's define in the `Datadog` module as well as `RUMTelemetry`. [The configuration is still reported at initialisation](https://github.com/DataDog/dd-sdk-ios/pull/1154/files#diff-7fae8d57fd7c8b0dc28f762018d1019d1f75530a064e083e7828d309e182be3dR299).

It's still unclear how we will tackle reporting configuration telemetry, but it will be tackled separately.